src/BedrockStreaming.php

```diff
@@ -44,6 +48,8 @@ public function getRules(): array
             'yoda_style' => false,
         ];
 
-        return $rules;
+        $newRules = \array_diff_key($this->extraRules, $standardRules);
+
+        return \array_merge($newRules, $standardRules);
```

Comment:
ℹ️ adding a new item to the `$standardRules` will be a potential breaking change for those who uses `$extraRules`

example: 

1. Toto uses `new BedrockStreaming(['explicit_string_variable' => true])`
2. `'explicit_string_variable' => false` is added to the `$standardRules` 
3. Toto updates its dependencies and then he cries because its code does not work anymore... Will it pin its dependencies to the previous version? 👀